docs/excel/excel-add-ins-ranges-advanced.md

```diff
@@ -350,8 +350,41 @@ Excel.run(function (context) {
 
 You can also find the cell responsible for spilling into a given cell by using the [Range.getSpillParent](/javascript/api/excel/excel.range#getspillparent--) method. Note that `getSpillParent` only works when the range object is a single cell. Calling `getSpillParent` on a range with multiple cells will result in an error being thrown (or a null range being returned for `Range.getSpillParentOrNullObject`).
 
+## Trace formula precedents
```

Comment:
Does "Trace" refer to the UI behavior of drawing the lines between formula cells and their precedents? If so, maybe we can rename this section to "Get formula precedents".

(Note that "trace" is used a couple times below, so if this gets changed, please change the other uses too.)

Reply:
Trace is the term on the UI button ("Trace precedents"), which on click draws arrows from precedent cells to the currently selected cell. I think it's important to identify in this section that the API is related Trace precedents button + behavior. Do you agree? I see your point about changing the title of the section, because this API doesn't draw arrows.

Reply:
While "trace" does likely refer to the drawing of arrows, I agree with your point that we should tie the API to the UI button. Do you think it would be worth noting in the text that our APIs don't draw any arrows?

Reply:
Yes, good point!